### DISCUSSION TITLE
insert into 使用.net 的MySqlConnector  大批量插入效率特别低下

### DISCUSSION THREAD

**Discussion by xyb213405** [apache/doris#30410] [RFC]
大概是600个字段的宽表  每次插入150个字段左右 使用如下所示的批量插入sql  一次5000条
`INSERT INTO example_tbl VALUES
(1000, "baidu1", 3.25)
(2000, "baidu2", 4.25)
(3000, "baidu3", 5.25);`
我是在客户端拼接好sql  然后使用MySqlConnector执行的  但是效率特别低下  5000条耗时在4秒左右
可能是什么问题造成的？
![image](https://github.com/apache/doris/assets/62739208/398ffefd-a0c0-47bc-8267-24154b415524)

**Comment by LemonLiTree** [comment]
推荐使用stream load

**Comment by zhbdesign** [comment]
你可以一个批次的量再更大一些，doris有事务，想非常快感觉有点难

**Comment by windowshk** [comment]
请问有解决方案吗？我这边也遇到这样的问题插入doris时超慢的。

**Reply by maruyue** [reply]
中文的问题，请到ask.selectdb.com上doris技术论坛提问。
